assets/static/js/dataset-submission.js

```diff
@@ -268,7 +268,17 @@ $(function() {
         // Populate dropdown with list of folders
         $.getJSON(url, function (data) {
             $.each(data, function (key, value) {
-                dropdown.append($('<option></option>').attr('value', value).text(key));
+                // remove trailing slash if exists from string.
+                value = value.replace(/\/$/, "");
+                let splitPath = value.toString().split("/");
+                dropdown.append($('<option></option>').attr('value', value).text(
+                    '/'
+                    + splitPath[splitPath.length - 3]
+                    + '/'
+                    + splitPath[splitPath.length - 2]
+                    + '/'
+                    + splitPath[splitPath.length - 1]
+                ));
```

Comment:
I think a regex looks cleaner than a split and concat.

```suggestion
                const regex = new RegExp('(?:.(?!\/.*\/.*\/))+$');
                let text = regex.exec(value)[0];
                dropdown.append($('<option></option>').attr('value', value).text(text));
```